src/redux/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducers from './books/books';
+import checkReducer from './categories/categories';
+
+const reducer = combineReducers({
+  booksReducers,
+  checkReducer,
+});
+
+const store = createStore(reducer, applyMiddleware(logger));
```

Comment:
- [ ] This module requires that `configureStore` from `@redux/toolkit` is used to create reducer. Therefore, kindly add `book` and `categories` to  `configureStore` method instead of `createstore`, similar to the example below:
```ruby
import `configureStore `from `@reduxjs/toolkit`

...

const store = configureStore({ reducer: { books: addRemoveReducer, category: categoriesReducer } });

...

```